body-pix/src/util.ts

```diff
@@ -71,10 +74,11 @@ function toInternalResolutionPercentage(
     return result;
   } else {
     tf.util.assert(
-        typeof internalResolution === 'number' && internalResolution < 1 &&
-            internalResolution > 0,
+        typeof internalResolution === 'number' &&
+            internalResolution <= MAX_INTERNAL_RESOLUTION &&
+            internalResolution >= MIN_INTERNAL_RESOLUTION,
         () =>
-            `inputResolution must be a string or number between 0 and 1, but ` +
+            `inputResolution must be a string or number between 0 and 4, but ` +
```

Comment:
this doesn't match the constant -- and let's use the constant in the error instead of hardcoding the string.